services/scan/src/precinct_scanner_app.ts

```diff
@@ -78,7 +46,7 @@ async function updateInterpreterConfig(
   if (!electionDefinition || !precinctSelection) {
     interpreter.unconfigure();
   } else {
-    const layouts = await loadLayouts(store);
+    const layouts = await store.loadLayouts();
```

Comment:
Now that layouts are cached, we can do all of this "configuring the interpreter" on each scan, instead of caching a reference to the interpreter. All of the rest of the data should be very quick to load from the db. This will remove the scariest part of this module - calling `updateInterpreterConfig` any time an API request modifies any of the interpreter args.

Currently we're in a bit of a halfway state where we have two layers of caching - one in the store, and then one in this file. Doesn't necessarily need to happen in this PR (and I'm happy to do it myself if you'd like, since I'll be working in this file soon), but I do think it will be a big win.

Thoughts?

Reply:
The `updateInterpreterConfig` thing is a real footgun and I agree, we should follow your suggestion. I'll try it here.

Reply:
I moved it in the latest commit, and tested to see that it adds an average of 5 milliseconds before each scan. Maybe we could speed that up by hitting the database fewer times. But does that seem acceptable?

Reply:
What's the overall time of the scan? How big of a percentage of that time is 5ms?

Reply:
The scan is a little more than 5 seconds for simple ballots, could be longer. So 5ms is like 0.1%. Seems OK?

Reply:
Yeah I'd say that's more than ok